.github/workflows/openbistest.yml

```diff
@@ -0,0 +1,47 @@
+name: interbis
```

Comment:
die Datei würde ich löschen
Hast du ja in den bestehenden workflow integriert

Reply:
gelöscht